truffle/src/main/java/org/jruby/truffle/nodes/coerce/ToAryNode.java

```diff
@@ -0,0 +1,58 @@
+package org.jruby.truffle.nodes.coerce;
+
+import com.oracle.truffle.api.CompilerDirectives;
+import com.oracle.truffle.api.dsl.NodeChild;
+import com.oracle.truffle.api.dsl.Specialization;
+import com.oracle.truffle.api.frame.VirtualFrame;
+import com.oracle.truffle.api.source.SourceSection;
+import org.jruby.truffle.nodes.RubyNode;
+import org.jruby.truffle.nodes.dispatch.CallDispatchHeadNode;
+import org.jruby.truffle.nodes.dispatch.DispatchHeadNodeFactory;
+import org.jruby.truffle.runtime.RubyContext;
+import org.jruby.truffle.runtime.control.RaiseException;
+import org.jruby.truffle.runtime.core.RubyArray;
+
+
+@NodeChild(value = "child", type = RubyNode.class)
+public abstract class ToAryNode extends RubyNode {
+
+    @Child
+    private CallDispatchHeadNode toAryNode;
```

Comment:
Minor style thing, but in every other case I've seen we have the annotation and the field declaration on the same line.